subprojects/core/src/integTest/groovy/org/gradle/api/tasks/ArchiveIntegrationTest.groovy

```diff
@@ -17,15 +17,15 @@ package org.gradle.api.tasks
 
 import org.apache.commons.lang.RandomStringUtils
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
-import org.gradle.integtests.fixtures.TestResources
+import org.gradle.integtests.fixtures.archives.TestReproducibleArchives
 import org.gradle.test.fixtures.archive.TarTestFixture
 import org.gradle.test.fixtures.file.TestFile
 import org.hamcrest.Matchers
-import org.junit.Rule
 import spock.lang.Issue
 
 import static org.hamcrest.Matchers.equalTo
 
+@TestReproducibleArchives
```

Comment:
I ignored it by using JUnits `assume` and adding a `reproducibleArchives` property at runtime.